### DISCUSSION TITLE
Is there an advantage to creating a default organization?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#61489] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I've been using GitHub for years, but only recently became aware of organizations. I thought I'd create a new organization, but it looks like I must first create an organization for the repos I've created so far, using the name of my GH account. Is there an advantage to creating an organization for all the repos I've got so far? Or doesn't it matter?

**Comment by GhostOf0days** [comment, marked as answer]
To be honest, only create organizations if it's an actual organization.
